src/App.js

```diff
@@ -1,26 +1,25 @@
-import './App.css';
-import {Recipes} from './components/recipes/recipes'
-import {GetRecipes} from './services/edamamApi.js'
+import "./App.css";
+import { Recipes } from "./components/recipes/recipes";
```

Comment:
since we have only one components to export in current files, i used the default export notation.